src/components/Planet/PlanetInfo/PlanetDescription/PlanetName/PlanetName.css

```diff
@@ -0,0 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Antonio:wght@700&display=swap');
```

Comment:
Fonts must be imported globally via CDN in index.html

Reply:
created a new task in trello for that